vowpalwabbit/parse_args.cc

```diff
@@ -1538,18 +1529,32 @@ options_i& load_header_merge_options(options_i& options, vw& all, io_buf& model)
   }
 
   if (count == 0 && saved_key != "") { options.insert(saved_key, ""); }
+}
+
+options_i& load_header_merge_options(options_i& options, vw& all, io_buf& model, bool& interactions_settings_doubled)
+{
+  std::string file_options;
+  save_load_header(all, model, true, false, file_options, options);
+
+  interactions_settings_doubled = check_interaction_settings_collision(options, file_options);
```

Comment:
doubled -> duplicated?

doubled and interactions makes me think that we want to double the interactions for some reason

Reply:
fixed